modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/list/List.tsx

```diff
@@ -150,7 +150,7 @@ export default function List(props: IFDSViewSectionProps) {
 			})
 		);
 
-		closeModal();
+		openDefaultSuccessToast();
```

Comment:
Setting toast as last allows us to do 
```js
await listVisualizationModePage.page.getByText('Success').waitFor();
```
The idea is, when the toast appears, `setListSections` is resolved and the field that we are about to check has new value. This is tiny bit scary to me, as React component needs to reload with updated state, but I didn't notice any flakiness. I am not sure if there is any cleaner approach to this.